.github/workflows/ci.yml

```diff
@@ -37,6 +37,7 @@ jobs:
             - 'shlr/rizin-shell-parser/**'
             - 'subprojects/**'
             - '.github/workflows/ci.yml'
+            - 'test/**'
```

Comment:
I think this change needs to be done in almost all ci files. See also appveyor please.

Reply:
Ok added the change to `.appveyor.yml` and `tcc.yml` 671e1dc. I don't think either `code-analysis.yml` or `newshell-treesitter-tests.yml` need this change.